collaboration/learning_goals.md

```diff
@@ -1,11 +1,36 @@
 # Learning Goals
 
+This project is designed to provide hands-on experience in collaborative
+group work, promoting a supportive and inclusive learning environment for all teammates
+We are setting these goals
+ to ensure that each one of us is growing
+ and gaining valuable experience and making the most
+ out of this project.
+
 ## Collective
 
-- Enhance collaboration by learning to communicate technical ideas clearly and respectfully.
+From this project, our aim is to gain hands-on experience working
+on group projects collaboratively. And enhance collaboration by learning to
+communicate technical ideas clearly and respectfully. We want to foster
+a healthy learning
+environment that provides support and help to every member in need.
+We aim to improve our technical capabilities like working with git and github,
+programming with Python, and problem solving and strategies.
+And develop healthy programming practices like using code review checklists and
+writing unit tests. And enhance collaboration
+by learning to communicate technical ideas clearly and respectfully.
 
 ## Individual
 
-- Learn to write more efficient and readable code.  
+- Learn to write more efficient and readable code that can be understood
+and used by any programmer regardless of it's complexity.
 - Gain confidence in contributing to team-based coding projects.
-- Learn python
+- Fully understand python basics and get more familiar with it by following guides
+and asking for help.
+- Practice on documenting and testing on different challenges and solutions.
+- Apply what we learned in the foundations track and get used to the workflows
+we were introduced to.
+- Get more familiar with Git commands, and gain experience interacting with the
+Command Line Interface to finish tasks rather than solely relying on the GUI.
+- Get used to GitHub's UI and how to interact with it and learn how
+to maximize it's capabilities.
```

Comment:
the file does not end with a newline character. To fix this issue, you can add a newline character at the end of the file.